README.md

```diff
@@ -11,10 +11,12 @@ cd crud-demo-app
 
 ## Setup the database
 
-Import ``config/schema/db.sql`` into your database.
-
 Modify ``config/app_local.php`` in the `Datasources` section to reflect your own database.
 
+Create the database schema `bin/cake migrations migrate`
+
+Seed the database `bin/cake migrations seed`
```

Comment:
```suggestion
Create the database schema `bin/cake migrations migrate`.
Seed the database `bin/cake migrations seed`.

You can also import the ``config/schema/db.sql`` file into your database instead of using the migrations.
```